composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": ">=5.6",
-        "illuminate/queue": "5.3.*",
+        "illuminate/queue": "5.3.16",
```

Comment:
should be `^5.3.16` I think

though who knows when the next breaking change is coming!